### PR TITLE
Intel(R) Optimized Caffe Training on AWS c5.18xlarge

### PR DESCRIPTION
**Pull request by daisyden**
The test is performed on 64 AWS c5.18xlarge nodes. Achieved top-5 accuracy 0.9307 within 6.17 hours with Resnet50. Please find the README.md and "Guide to Launch Intel Caffe Multi Nodes Training on AWS.docx" under https://github.com/intel/caffe/blob/DAWN_Bench for detailed steps to reproduce the result.

### CODE REVIEW

File: ImageNet/train/Intel_Resnet50_ec2_Xeon_8124M_intelcaffe.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    "author": "Intel(R) Corporation",
+    "authorEmail": "daisy.deng@intel.com",
+    "framework": "Intel(R) Optimized Caffe",
+    "codeURL": "https://github.com/intel/caffe/tree/DAWN_Bench",
+    "model": "ResNet50",
+    "hardware": "64 nodes with Xeon Platinum 8124M / 144 GB / 36 Cores (Amazon EC2 [c5.18xlarge])",
+    "costPerHour": 195.84,
+    "timestamp": "2018-04-18",
+    "logFilename": "https://github.com/intel/caffe/tree/DAWN_Bench/models/intel_optimized_models/multinode/resnet_50_64_nodes_2k_batch/outputCluster-skx-64.txt.tgz",
+    "misc": {
```
Comment (codyaustun) [inline ImageNet/train/Intel_Resnet50_ec2_Xeon_8124M_intelcaffe.json:12]: Can you add one more field to `misc` with a description of the communication method you used? Even a link to https://github.com/intel/caffe/wiki/Multinode-guide#communication from your README would be fine.

File: ImageNet/train/Intel_Resnet50_ec2_Xeon_8124M_intelcaffe.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+        "Intel(R) Optimized Caffe": "https://github.com/intel/caffe/tree/DAWN_Bench, commit abf2a5c",
+        "Intel(R) MKLDNN": "464c268e",
+        "Data Set": "please use raw lmdb for inference, to create raw lmdb you can use 'sh examples/imagenet/create_imagenet_raw.sh",
+        "optimizer": "SGD with Momentum",
+        "momentum": 0.9,
+        "weightDecay": 0.0001,
+        "batchSize": 2048,
+        "warmUp": "the learning rate starts at 0.1, then it increases linearly per iteration for the first 5 epochs",
```
Comment (codyaustun) [inline ImageNet/train/Intel_Resnet50_ec2_Xeon_8124M_intelcaffe.json]: You say you do a 5 epoch warm up here, but the schedule you have below looks like it only has 3 epochs for the warm up. Please update to make them consistent. Specifically, I think the 5 epochs here should be 3 to match the length of your TSV.

**Review by codyaustun — CHANGES_REQUESTED** [review]
Minor issues, but overall looks good.

**Comment by daisyden** [comment]
@codyaustun I have updated according to your comments. Thanks a lot!

**Review by codyaustun — APPROVED** [review]

**Pull request merged**